.vib/moodle/goss/goss.yaml

```diff
@@ -0,0 +1,9 @@
+file:
+  /bitnami/moodledata:
+    exists: true
+    filetype: directory
+    mode: "2775"
+http:
+  https://moodle:{{ .Vars.service.ports.https }}:
+    status: 200
+    allow-insecure: true
```

Comment:
You are missing the test for the `securityContext` parameters.

Reply:
@mdhont you missed adding the `command` like for the `check-user-info` test.

Reply:
I've added it now